### ISSUE TITLE
Error with fotmob. `fotmob_get_stat_and_season_options`

### ISSUE THREAD

**Issue by visvaya** [JaseZiv/worldfootballR#261]
Hi, today I encountered an error while trying to get data from fotmob using `fotmob_get_season_stats`.
A few days ago it worked without any problems. Do you know if there have been any changes on the fotmob website or am I doing something wrong? I would appreciate if you take a look at this issue @tonyelhabr

---
```r
library(worldfootballR)
library(dplyr)
#> 
#> Dołączanie pakietu: 'dplyr'
#> Następujące obiekty zostały zakryte z 'package:stats':
#> 
#>     filter, lag
#> Następujące obiekty zostały zakryte z 'package:base':
#> 
#>     intersect, setdiff, setequal, union
library(tidyr)
library(reprex)

ekstraklasa_stats_fotmob <- fotmob_get_season_stats(
  country = "POL",
  league_name = "Ekstraklasa",
  season_name = "2022/2023",
  team_or_player = "player",
  stat_name = "Accurate long balls per 90"
)
#> Defaulting `season` to latest ("2022/2023").
#> This message is displayed once per session.
#> Warning in sprintf("Can't find season stats data. Failed with the following error:\n", : one argument not used by format 'Can't find season stats data. Failed with the following error:
#> '
#> Error in `map2()`:
#> ℹ In index: 1.
#> Caused by error in `pmap()`:
#> ℹ In index: 1.
#> Caused by error in `.fotmob_get_stat_and_season_options()`:
#> ! Can't find season stats data. Failed with the following error:
#> Can't find season stats data. Failed with the following error:
#> Backtrace:
#>      ▆
#>   1. ├─worldfootballR::fotmob_get_season_stats(...)
#>   2. │ └─purrr::map2_dfr(...)
#>   3. │   └─purrr::map2(.x, .y, .f, ...)
#>   4. │     └─purrr:::map2_("list", .x, .y, .f, ..., .progress = .progress)
#>   5. │       ├─purrr:::with_indexed_errors(...)
#>   6. │       │ └─base::withCallingHandlers(...)
#>   7. │       ├─purrr:::call_with_cleanup(...)
#>   8. │       └─worldfootballR (local) .f(.x[[i]], .y[[i]], ...)
#>   9. │         └─purrr::pmap_dfr(list(.x, .y, urls$id), ~g(..1, ..2, ..3))
#>  10. │           └─purrr::pmap(.l, .f, ...)
#>  11. │             └─purrr:::pmap_("list", .l, .f, ..., .progress = .progress)
#>  12. │               ├─purrr:::with_indexed_errors(...)
#>  13. │               │ └─base::withCallingHandlers(...)
#>  14. │               ├─purrr:::call_with_cleanup(...)
#>  15. │               └─worldfootballR (local) .f(.l[[1L]][[i]], .l[[2L]][[i]], .l[[3L]][[i]], ...)
#>  16. │                 └─worldfootballR (local) g(..1, ..2, ..3)
#>  17. │                   └─worldfootballR:::.fotmob_get_stat_and_season_options(...)
#>  18. │                     └─base::stop(...)
#>  19. └─base::.handleSimpleError(...)
#>  20.   └─purrr (local) h(simpleError(msg, call))
#>  21.     └─cli::cli_abort(...)
#>  22.       └─rlang::abort(...)

packageVersion("worldfootballR")
#> [1] '0.6.2.9400'

sessionInfo()
#> R version 4.2.3 (2023-03-15 ucrt)
#> Platform: x86_64-w64-mingw32/x64 (64-bit)
#> Running under: Windows 10 x64 (build 22000)
#> 
#> Matrix products: default
#> 
#> locale:
#> [1] LC_COLLATE=Polish_Poland.utf8  LC_CTYPE=Polish_Poland.utf8   
#> [3] LC_MONETARY=Polish_Poland.utf8 LC_NUMERIC=C                  
#> [5] LC_TIME=Polish_Poland.utf8    
#> 
#> attached base packages:
#> [1] stats     graphics  grDevices utils     datasets  methods   base     
#> 
#> other attached packages:
#> [1] reprex_2.0.2              tidyr_1.3.0              
#> [3] dplyr_1.1.1               worldfootballR_0.6.2.9400
#> 
#> loaded via a namespace (and not attached):
#>  [1] pillar_1.9.0      compiler_4.2.3    prettyunits_1.1.1 tools_4.2.3      
#>  [5] progress_1.2.2    digest_0.6.31     lubridate_1.9.2   jsonlite_1.8.4   
#>  [9] evaluate_0.20     lifecycle_1.0.3   tibble_3.2.1      timechange_0.2.0 
#> [13] pkgconfig_2.0.3   rlang_1.1.0       cli_3.6.1         rstudioapi_0.14  
#> [17] curl_5.0.0        yaml_2.3.7        xfun_0.38         fastmap_1.1.1    
#> [21] xml2_1.3.3        withr_2.5.0       httr_1.4.5        stringr_1.5.0    
#> [25] knitr_1.42        janitor_2.2.0     hms_1.1.3         generics_0.1.3   
#> [29] fs_1.6.1          vctrs_0.6.1       tidyselect_1.2.0  glue_1.6.2       
#> [33] snakecase_0.11.0  R6_2.5.1          fansi_1.0.4       rmarkdown_2.21   
#> [37] selectr_0.4-2     tzdb_0.3.0        readr_2.1.4       purrr_1.0.1      
#> [41] magrittr_2.0.3    htmltools_0.5.5   rvest_1.0.3       utf8_1.2.3       
#> [45] stringi_1.7.12    crayon_1.5.2
```

**Comment by tonyelhabr** [comment]
hmm ok I'll take a look

**Comment by tonyelhabr** [comment]
addressed with #263
